tests/main/ubuntu-core-classic/task.yaml

```diff
@@ -13,14 +13,14 @@ restore: |
 execute: |
     echo "Ensure classic can be installed"
     snap install --devmode --beta classic
-    snap list|grep classic
+    snap list|MATCH classic
 
     echo "Check that classic can run commands inside classic"
     classic test -f /var/lib/dpkg/status
 
     echo "Ensure that after classic exits no processes are left behind"
     classic "sleep 133713371337&"
-    if ps afx|grep 133713371337|grep -v grep; then
+    if ps afx|grep '[1]33713371337'; then
```

Comment:
it's a very common idiom that I didn't think needed explaining (but obviously i was wrong). Because `[f]oo` matches `foo` but not `[f]oo`, it avoids having to prune the grep you're using to search for a process from the process list. (try it: `ps ax | grep bash` vs `ps ax | grep [b]ash`).